cafebabel/templates/users/detail.html

```diff
@@ -43,9 +48,20 @@ <h2>About</h2>
 </section>
 
 <section>
-  <h2>Contributions</h2>
-  {% for article in articles %}
-    {{ card(article) }}
-  {% endfor %}
+  <h2 class="contribution-switcher article-switcher active">Articles</h2><h2 class="contribution-switcher translation-switcher">Translations</h2>
```

Comment:
Is that possible to put the Translations title on top of the `translation-list` to remain consistent HTML wise? And then adapt with CSS to keep the same look.